libs/expo/betterangels/src/lib/screens/AddEditClient/ImportantNotes/index.tsx

```diff
@@ -0,0 +1,52 @@
+import { Spacings } from '@monorepo/expo/shared/static';
+import { Accordion, Textarea } from '@monorepo/expo/shared/ui-components';
+import { RefObject } from 'react';
+import { useFormContext } from 'react-hook-form';
+import { ScrollView, View } from 'react-native';
+import {
+  CreateClientProfileInput,
+  UpdateClientProfileInput,
+} from '../../../apollo';
+
+interface IImportantNotesProps {
+  expanded: undefined | string | null;
+  setExpanded: (expanded: undefined | string | null) => void;
+  scrollRef: RefObject<ScrollView>;
+}
+
+export default function ImportantNotes(props: IImportantNotesProps) {
+  const { scrollRef, expanded, setExpanded } = props;
+  const { control } = useFormContext<
+    UpdateClientProfileInput | CreateClientProfileInput
+  >();
+
+  const isImportantNotes = expanded === 'Important Notes';
+  return (
+    <Accordion
+      scrollRef={scrollRef}
+      expanded={expanded}
+      setExpanded={() => {
+        setExpanded(isImportantNotes ? null : 'Important Notes');
```

Comment:
The `Important Notes` is being repeated (we check for it in the `isImportantNotes` const). 
Can we pull it out into a const, like `accordionKey` or something? The name is less important, but a typo can easily break this.

Reply:
good one. We have a code cleaning task can walk through this as well